app/models/fhir/domain_resource.rb

```diff
@@ -9,9 +9,9 @@ class DomainResource < Resource
 
     def self.transform_json(json_hash, target = DomainResource.new)
       result = self.superclass.transform_json(json_hash, target)
-      result['text'] = Narrative.transform_json(json_hash['text']) unless json_hash['text'].nil?      
+      result['text'] = Narrative.transform_json(json_hash['text']) unless json_hash['text'].nil?
```

Comment:
hm.. isn't it something generated automatically? I assume it will be overridden

Reply:
Correct. The generator will get updated as part of another story. This was just a quick workaround.

Reply:
Created [MAT-1337](https://jira.cms.gov/browse/MAT-1337) to investigate and resolve this issue in the generator.